src/TokenStreamRewriter.ts

```diff
@@ -137,11 +137,11 @@ export class TokenStreamRewriter {
 		this.rollback(TokenStreamRewriter.MIN_TOKEN_INDEX, programName);
 	}
 
-	public insertAfter(t: Token, text: any): void;
-	public insertAfter(index: number, text: any): void;
-	public insertAfter(t: Token, text: any, programName: string): void;
-	public insertAfter(index: number, text: any, programName: string): void;
-	public insertAfter(tokenOrIndex: Token | number, text: any, programName: string = TokenStreamRewriter.DEFAULT_PROGRAM_NAME): void {
+	public insertAfter(t: Token, text: {}): void;
```

Comment:
@sharwell, declaring a formal parameter named `text` to be an `object` just seems weird.    If this is intentional, perhaps a comment is called for.

Reply:
text is actually an `Object` here, and strings can be passed to this method. The rendering pipeline just calls `toString()` on the replacement values.